bin/lock

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bin/ruby
```

Comment:
well, so everything under `bin` isn't copied over to our release distributable? that is my main concern I guess